pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -42,7 +42,7 @@
 
     public static final String LEAD_CONTROLLER_RESOURCE_ENABLED_KEY = "RESOURCE_ENABLED";
     public static final String ENABLE_CASE_INSENSITIVE_PQL_KEY = "enable.case.insensitive.pql";
-
+    public static final String ENABLE_BROKER_QUERY_LIMIT_OVERRIDE_KEY = "enable.broker.query.limit.override";
```

Comment:
Can you please move this under class `Broker` instead of `Helix`? Also, I would recommend changing it to `CONFIG_OF_ENABLE_QUERY_LIMIT_OVERRIDE` to be consistent with other configs.